frontend/app/routes/hr-advisor/request/submission-details.tsx

```diff
@@ -163,15 +162,15 @@ export async function loader({ context, params, request }: Route.LoaderArgs) {
   return {
     documentTitle: t('app:submission-details.page-title'),
     defaultValues: {
-      submitter: requestData.submitter, // The Hr-advisor can't create a Request. The submitter information is coming from the saved Request.
-      hiringManager: requestData.hiringManager,
-      subDelegatedManager: requestData.subDelegatedManager,
-      hrAdvisor: requestData.hrAdvisor,
-      workUnit: requestData.workUnit,
-      languageOfCorrespondence: requestData.languageOfCorrespondence,
-      additionalComment: requestData.additionalComment,
-      additionalContact: requestData.additionalContact,
-      alternateContactEmailAddress: requestData.alternateContactEmailAddress,
+      'submitter': requestData.submitter, // The Hr-advisor can't create a Request. The submitter information is coming from the saved Request.
+      'hiringManager': requestData.hiringManager,
+      'subDelegatedManager': requestData.subDelegatedManager,
+      'hrAdvisor': requestData.hrAdvisor,
+      'workUnit': requestData.workUnit,
+      'languageOfCorrespondence': requestData.languageOfCorrespondence,
+      'additionalComment': requestData.additionalComment,
+      'additionalContact': requestData.additionalContact,
+      'additionalContact.businessEmailAddress': requestData.additionalContact?.businessEmailAddress,
```

Comment:
We are already sending the additionalContact object that would contain businessEmailAddress, not sure why we need to send additionalContact.businessEmailAddress specifically?
```suggestion
       submitter: requestData.submitter, // The Hr-advisor can't create a Request. The submitter information is coming from the saved Request.
      hiringManager: requestData.hiringManager,
      subDelegatedManager: requestData.subDelegatedManager,
      hrAdvisor: requestData.hrAdvisor,
      workUnit: requestData.workUnit,
      languageOfCorrespondence: requestData.languageOfCorrespondence,
      additionalComment: requestData.additionalComment,
      additionalContact: requestData.additionalContact,
```